lib/scenic/adapters/postgres.rb

```diff
@@ -15,6 +18,14 @@ module Adapters
     #
     # @api extension
     class Postgres
+      # Creates an instance of the Scenic Postgres adapter
+      #
+      # @param connection The database connection the adapter should use. This
+      #   defaults to `ActiveRecord::Base.connection`
+      def initialize(connection = ActiveRecord::Base.connection)
```

Comment:
It's used in migrations and schema dumping, so not really. We expect to have an connection established by this point.